its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1449,6 +1443,11 @@ public void didChangeTaintVulnerabilities(String configurationScopeId, Set<UUID>
         didChangeTaintVulnerabilitiesEvents
           .add(new DidChangeTaintVulnerabilitiesParams(configurationScopeId, closedTaintVulnerabilityIds, addedTaintVulnerabilities, updatedTaintVulnerabilities));
       }
+
+      @Override
+      public void didSynchronizeConfigurationScopes(DidSynchronizeConfigurationScopeParams params) {
+        didSynchronizeConfigurationScopes.add(params.getConfigurationScopeIds().toString());
```

Comment:
This look suspicious. Shouldn't it be 
`didSynchronizeConfigurationScopes.addAll(params.getConfigurationScopeIds());`

Reply:
well spotted, that was leftover from a quick attack on the problem